cmd/pow/cmd/prepare/prepare.go

```diff
@@ -248,64 +266,99 @@ The piece-size and piece-cid are printed to stderr. For scripting usage, its rec
 		if errCommP != nil {
 			c.Fatal(fmt.Errorf("calculating piece-size and PieceCID: %s", err))
 		}
+
+		if aggregate {
+			rootNd, err := dagService.Get(ctx, payloadCid)
+			if err != nil {
+				c.Fatal(fmt.Errorf("get root node: %s", err))
+			}
+			manifestLnk := rootNd.Links()[0]
+			manifestNd, err := dagService.Get(ctx, manifestLnk.Cid)
+			if err != nil {
+				c.Fatal(fmt.Errorf("get manifest node: %s", err))
+			}
+			manifestF, err := unixfile.NewUnixfsFile(context.Background(), dagService, manifestNd)
+			if err != nil {
+				c.Fatal(fmt.Errorf("get unifxfs manifest file: %s", err))
+			}
+			manifest := manifestF.(files.File)
+			fManifest, err := os.Create(args[1] + ".manifest")
+			if err != nil {
+				c.Fatal(fmt.Errorf("creating manifest file: %s", err))
+
+			}
+			defer func() {
+				if err := fManifest.Close(); err != nil {
+					c.Fatal(fmt.Errorf("closing manifest file: %s", err))
+				}
+			}()
+			if _, err := io.Copy(fManifest, manifest); err != nil {
+				c.Fatal(fmt.Errorf("writing manifest file: %s", err))
+			}
```

Comment:
This code section is to retrieve from the generated batch the manifest file that lives inside.
The manifest file lives as the first file in the root folder (defined in the spec), thus `Links()[0]` on 275 will always succeed and give us the file.

The rest is pulling the file and writing it to the current folder.